src/pages/Home/HomePageInternal.tsx

```diff
@@ -1,7 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-function HomePageInternal(): JSX.Element {
-  return <div>Internal Home Page</div>;
-}
+import ClassNamesLogic from '../../common/components/Util/ClassNamesLogic';
+import ClubsTab from './components/ClubsTab';
```

Comment:
Let's see what the team decides on Discord, but we should have uniform naming across UX + code, either 'community' or a 'club', but if we use both it'll be messy.

Reply:
I am following the design jason made, we ll change later if needed

Reply:
I asked team-general on Discord and we decided to use "community", so let's use that everywhere.